chain/data.go

```diff
@@ -112,6 +112,8 @@ type SpecData struct {
 	//
 	// InactivityPenaltyQuotient is the inactivity penalty quotient.
 	InactivityPenaltyQuotient uint64 `mapstructure:"inactivity-penalty-quotient"`
+	// InactivityPenaltyQuotientAltair is the inactivity penalty quotient for Altair.
+	InactivityPenaltyQuotientAltair uint64 `mapstructure:"inactivity-penalty-quotient-altair"`
```

Comment:
not sure adding configs we will never use only to serve APIs responses is the right approach.
Wouldn't it be better to configure this value only where needed?
cc @calbera

Reply:
In agreement. @nidhi-singh02 Let's remove any changes to chain/ and config/spec/ in this PR. These values can be hardcoded as placeholders as needed. As discussed this is only temporary until we properly implement and use inactivityPenaltyQuotients

Reply:
I have removed changes to chain/ and config/spec.  And hardcoded these values in node-api.